### ISSUE TITLE
Explain why there are both FTB and FTB Legacy and go over the differences between them

### ISSUE THREAD

**Issue by Poopooracoocoo** [PrismLauncher/prismlauncher.org#253]

**Comment by lolouthefox** [comment]
If you mean like explaining on the website FTB is going to disappear from Prism. And technically if you want to install FTB You know which version you want

**Comment by GameLord2011** [comment]
No, they meant explain the difference between the instance creation options that use the modpacks.ch backend vs. the legacy api for FTB.
